CMakeLists.txt

```diff
@@ -263,5 +279,6 @@ if (TESTS_BUILD)
 endif ()
 
 if (${VULKAN_HPP_INSTALL})
```

Comment:
I'm not sure why the install target is behind a feature guard. The install target is not executed when building `ALL` only when explicitly requested